spec/features/posting_peep_spec.rb

```diff
@@ -0,0 +1,7 @@
+feature 'Posting a peep' do
+  scenario 'User can post and view their peep' do
+    visit('/')
+    fill_in('peep', with: 'Hello World!')
+    expect(page).to have_content('Hello World!')
+  end
+end
```

Comment:
Another feature test you could potentially add here is testing that the homepage is viewable. This will test that the user can load the web app and they'll see everything they need in order to post their first Peep.